src/engine/renderer/tr_init.cpp

```diff
@@ -880,6 +880,14 @@ ScreenshotCmd screenshotPNGRegistration("screenshotPNG", ssFormat_t::SSF_PNG, "p
 			"fullscreen"
 		};
 
+		Log::Notice( "%sOpenGL driver vendor: %s",
+			Color::ToString( Util::ordinal(glConfig2.driverVendor) ? Color::Green : Color::Yellow ),
```

Comment:
Compare to UNKNOWN instead of using the numeric value

Reply:
I loved the way it was shorter! 😏️

Reply:
In fact thanks to the various `return`, this was even the only line requiring to test for `UNKNOWN`.